tools/cron/common.py

```diff
@@ -626,11 +627,17 @@ def _run_test(config):
 
         if not test_cases_completed and not os.path.exists(results_file_path):
             if timedelta(seconds=current_time - last_check_time) > timedelta(seconds=timeguard):
+                if startup_fail_count == 0:
+                    log(f"Test run has not been started on time. No more retries...")
```

Comment:
```suggestion
                    log("Test run has not been started on time. No more retries...")
```

The `f` is superfluous if not doing any formatting